tests/test-backend-ops.cpp

```diff
@@ -56,7 +56,7 @@ static void init_tensor_uniform(ggml_tensor * tensor, float min = -1.0f, float m
         GGML_ASSERT(size % ggml_blck_size(tensor->type) == 0);
         std::vector<uint8_t> dataq(ggml_row_size(tensor->type, size));
         int64_t hist[16];
-        ggml_quantize_chunk(tensor->type, data.data(), dataq.data(), 0, size, hist);
+        ggml_quantize_chunk(tensor->type, data.data(), dataq.data(), 0, size/tensor->ne[0], tensor->ne[0], hist, nullptr);
```

Comment:
I guess `test-backend-ops` could generate a dummy importance matrix (say full of `1.0f`) just to be able to perform the 2-but quantizations - even if the values would not make sense. For backend comparisons it should be enough. This way we will be able to add the `IQ2` quants to the tests.

We can do this later